quiz.js

```diff
@@ -1,168 +1,137 @@
-        //emptys right/wrong area
-        rightWrongEl.empty();
-
-        cheer.pause();
-        jeer.pause();
-        function fix (errorString){
-            if(errorString.includes("&#039;")){
-                console.log("its in there")
-            }
-            else {(errorString.includes("&#039;"))
-                    console.log("its not there")
-            }
-            while(errorString.includes("&#039;")){
-                errorString = errorString.replace("&#039;", "\'")
-                }
-            while(errorString.includes("&quot;")){
-                errorString = errorString.replace("&quot;","\"")
-                }
-            while(errorString.includes("&amp;")){
-                errorString = errorString.replace("&amp;","&")
-            }
-            while(errorString.includes("&Uuml;")){
-                errorString = errorString.replace("&Uuml;","Ü")
-            }    
-            while(errorString.includes("&eacute;")){
-                errorString = errorString.replace("&eacute;","é")
-            }        
-            return errorString
-        }
-        //sets currentQuestion equal to questions index
+        resetQuestionZone();        
+        
         let currentQuestion = questions[questionIndex];
-        //set correctAnswer equal to the correct answer of the current question
-        correctAnswer = currentQuestion.correct_answer;
-        //tells user what question number they are on
+        correctAnswer       = fix(currentQuestion.correct_answer);
+
         questionNumberEl.text(`Question: ${questionIndex + 1} / 10`)
-        //tells user score
-        scoreEl.text(`Score: ${score}`);
-        //shows user the current question after a fix
-        let fixedCurrentQuestion = fix(currentQuestion.question)
-        questionEl.text(`Question: ${fixedCurrentQuestion}`);
-        //shows user what category the current question is from
-        categoryEl.text(`Category: ${currentQuestion.category}`);
-        //sets up array of all possible answer choices using the spread operator for incorrect answers
-        console.log(currentQuestion.incorrect_answers)
-        for (var i = 0; i < 2; i++) {
-            currentQuestion.incorrect_answers[i] = fix(currentQuestion.incorrect_answers[i]);
-          }
-        // let fixedIncorrectAnswers = fix(currentQuestion.incorrect_answers)
-        let fixedCorrectAnswer = fix(currentQuestion.correct_answer)
-        let questionChoices = [...currentQuestion.incorrect_answers, fixedCorrectAnswer]
-        //shuffle array randomly
-        let randomQuestionChoices = questionChoices.sort(() => Math.random() - 0.5);
-        //creates a temporary btn for each of the choices and adds them to the answer zone; using jQuery adds text, value, class, & onclick event to each btn
-        randomQuestionChoices.forEach(function (value) {
-            let tempBtn = $('<button>').text(value).val(value).addClass('hollow button answer').click(validateAnswer);
```

Comment:
I really like these chained calls! Unfortunately, I don't have as wide a monitor as Josh, so that's why I put each successive call on a new line to see it better 👍